Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Unreleased
 
+# v0.16.0
+- [changed] Implemented LRU garbage collection for on-disk persistence. The SDK
```

Comment:
I'd suggest a phrasing that more actively describes that users will be affected by this unless they take action. Also, "LRU" is perhaps more techie than we should use in the headline? How about something like:

Added a garbage collection process for older documents in on-disk persistence.

Reply:
+1 to this, and minor tweak for clarity/flow:

Added a garbage collection process to on-disk persistence that removes older documents.

Reply:
Can even emphasize impact by saying something like "automatically removes older documents."